.azure-pipelines/templates/package-binaries.yml

```
@@ -3,11 +3,19 @@ parameters:
     default: linux
   - name: zipcommand
     default: "zip -9rv"
+  - name: useMinGW
+    default: false
```

May have to explicitly specify the type:
```suggestion
  - name: useMinGW
    default: false
    type: boolean
```